src/Card.js

```diff
@@ -0,0 +1,13 @@
+// Each card needs an id, question, and possible answers
+// Data file gives properties of id, question, answers, correct answer
```

Comment:
Some pseudo code to to better understand what I need to do before creating a card class. I will make sure to delete this when I refactor and clean up my files.